astrocut/cube_cut.py

```diff
@@ -457,17 +463,102 @@ def _update_primary_header(self, primary_header):
         # Adding cutout specific headers
         primary_header['CREATOR'] = ('astrocut', 'software used to produce this file')
         primary_header['PROCVER'] = (__version__, 'software version')
+        primary_header['FFI_TYPE'] = (product, 'the FFI type used to make the cutouts')
+        # TODO : The name of FIRST_FFI (and LAST_FFI) is too long to be a header kwd value.
+        # Find a way to include these in the headers without breaking astropy (maybe abbreviate?).
+        #primary_header['FIRST_FFI'] = (self.first_ffi, 'the FFI used for the primary header keyword values, except TSTOP')
+        #primary_header['LAST_FFI'] = (self.last_ffi, 'the FFI used for the TSTOP keyword value')
 
         primary_header['RA_OBJ'] = (self.center_coord.ra.deg, '[deg] right ascension')
         primary_header['DEC_OBJ'] = (self.center_coord.dec.deg, '[deg] declination')
 
-        primary_header['TIMEREF'] = ('SOLARSYSTEM', 'barycentric correction applied to times')        
-        primary_header['TASSIGN'] = ('SPACECRAFT', 'where time is assigned')                         
-        primary_header['TIMESYS'] = ('TDB', 'time system is Barycentric Dynamical Time (TDB)')
+        timeref = 'SOLARSYSTEM' if product == 'SPOC' else 'N/A'
+        tassign = 'SPACECRAFT' if product == 'SPOC' else 'N/A'
+        primary_header['TIMEREF'] = (timeref, 'barycentric correction applied to times')        
+        primary_header['TASSIGN'] = (tassign, 'where time is assigned')
+
+        timesys = 'TDB' if product == 'SPOC' else 'TDT'
+        timesys_desc = 'Barycentric' if product == 'SPOC' else 'Terrestrial'
+        primary_header['TIMESYS'] = (f'{timesys}', f'time system is {timesys_desc} Dynamical Time ({timesys})')
+
         primary_header['BJDREFI'] = (2457000, 'integer part of BTJD reference date')           
         primary_header['BJDREFF'] = (0.00000000, 'fraction of the day in BTJD reference date')    
         primary_header['TIMEUNIT'] = ('d', 'time unit for TIME, TSTART and TSTOP')
 
+        if product == 'TICA':
+
+            # Adding some missing kwds not in TICA (but in Ames-produced SPOC ffis)
```

Comment:
Is it worth adding some tests for the header manipulations?

Reply:
Would it make sense to read in keywords to be added and deleted from a config file?